src/chains/ethereum/ethereum/src/database.ts

```diff
@@ -70,31 +87,28 @@ export default class Database extends Emittery {
       }
       this.directory = directory;
 
-      // specify an empty `prefix` for browser-based leveldown (level-js)
-      const leveldownOpts = { prefix: "" };
-      const store = encode(leveldown(directory, leveldownOpts), levelupOptions);
-      this.#rootStore = store;
-      db = levelup(store);
+      const store = <GanacheLevel>new Level(directory, LEVEL_OPTIONS);
```

Comment:
```suggestion
      const store = new Level(directory, LEVEL_OPTIONS) as GanacheLevel;
```
For consistencies sake